Add documentation for VS C++ Redist (#20399)
dotnet/AspNetCore.Docs: aspnetcore/host-and-deploy/iis/hosting-bundle.md

>
> If the Hosting Bundle is installed after installing the 64-bit (x64) version of .NET Core, SDKs might appear to be missing ([No .NET Core SDKs were detected](xref:test/troubleshoot#no-net-core-sdks-were-detected)). To resolve the problem, see <xref:test/troubleshoot#missing-sdk-after-installing-the-net-core-hosting-bundle>.

## Direct download (current version)

Download the installer using the following link:

[Current .NET Core Hosting Bundle installer (direct download)](https://dotnet.microsoft.com/permalink/dotnetcore-current-windows-runtime-bundle-installer)

## Visual C++ Redistributable Requirement

> On older versions of Windows, for example Windows Server 2012 R2, you'll need to install the Visual Studio C++ 2015, 2017, 2019 Redistributable. Otherwise, a confusing error messaging saying: `The data is the error.` will be output in the Windows Event Log.

Reviewer: Drops future tense ...

```suggestion
On older versions of Windows, for example Windows Server 2012 R2, install the Visual Studio C++ 2015, 2017, 2019 Redistributable. Otherwise, a confusing error message in the Windows Event Log reports that `The data is the error.`
```

... might need to code-fence **Visual Studio C++ 2015, 2017, 2019 Redistributable**, too, because MT might over-translate and make that into something mangled. Same thing for the link text below.

> [!WARNING]
> Some installers contain release versions that have reached their end of life (EOL) and are no longer supported by Microsoft. For more information, see the [support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).

## Options

> 1.  The following parameters are available when running the installer from an administrator command shell:

Reviewer: Drop space after the period ...

```suggestion
1. The following parameters are available when running the installer from an administrator command shell:
```

   * `OPT_NO_SHAREDFX=1`: Skip installing the ASP.NET Shared Framework (ASP.NET runtime). Used when the server only hosts [self-contained deployments (SCD)](/dotnet/core/deploying/#self-contained-deployments-scd).
   * `OPT_NO_X86=1`: Skip installing x86 runtimes. Use this parameter when you know that you won't be hosting 32-bit apps. If there's any chance that you will host both 32-bit and 64-bit apps in the future, don't use this parameter and install both runtimes.
   * `OPT_NO_SHARED_CONFIG_CHECK=1`: Disable the check for using an IIS Shared Configuration when the shared configuration (`applicationHost.config`) is on the same machine as the IIS installation. *Only available for ASP.NET Core 2.2 or later Hosting Bundler installers.* For more information, see <xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration>.

> [!NOTE]
> For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).

> ## Restarting IIS

Reviewer: Can this just say "restart" because we've been cautioned about gerunds in section headings ...

```suggestion
## Restart IIS
```

> On install, the hosting bundle may require IIS to restart. For example, this can be due to `dotnet` not being on the PATH for already running IIS worker processes.

Reviewer: 🤔 I think we were going with the proper noun everywhere back in the day. We *might* be able to avoid the state verb here, too. See if something like this works (and you can edit this comment directly, change the suggestion, then add it to the batch ... forgive me, you probably already know that 😉) ...

```suggestion
On install, the Hosting Bundle may require a manual IIS restart. For example, the `dotnet` CLI tooling (command) might not exist on the PATH for running IIS worker processes.
```

> To fix this, you may stop and start IIS by running this in an elevated commandline:

Reviewer: ```suggestion
To manually stop and start IIS, execute the following commands in an elevated command shell:
```
